winui/Ribbon/RibbonScreenTip.md

```diff
@@ -0,0 +1,232 @@
+---
+layout: post
+title: ScreenTip | WinUI | Ribbon | Syncfusion
+description: Learn more about the ScreenTip in the Syncfusion WinUI Ribbon (SfRibbon) control.
+platform: winui
+control: Ribbon
+documentation: ug
+---
+
+# ScreenTip in WinUI Ribbon Control
+
+`ScreenTip` is an enhanced ToolTip and it is used to show a popup window when the pointer hovers over a ribbon item. The pop-up provides helpful details about the function of the ribbon item.
+
+The ScreenTip consist of two parts 
+    
+* **Title:** A property of a type object that specifes the title of the ScreenTip.
+* **Content:** A property of a type object that specifes the content of the ScreenTip.
+

```

Comment:
The ScreenTip has two properties.

* **Title:** Specifies the title of the ScreenTip.
* **Content:** Specifies the content of the ScreenTip.

Reply:
Updated